### PR TITLE
Add functionality to install modules during init

### PR DESCRIPTION
**Pull request by Luke-Roy-IBM**
Install modules specified in package.json during initialisation as a alternative to packageing them. Detect if the node_modules directory is  missing and installes the node modules.

### CODE REVIEW

File: core/nodejsActionBase/runner.js [hunk 1]
```diff
@@ -46,6 +46,17 @@ function initializeActionHandler(message) {
                     return Promise.reject('Zipped actions must contain either package.json or index.js at the root.');
                 }
 
+                // install npm modules during init if source code zip doesn´t containt them
```
Comment (style95) [inline core/nodejsActionBase/runner.js:49]: This is nice!

Comment (Luke-Roy-IBM) [reply core/nodejsActionBase/runner.js:49]: From my tests so far the current functionality is not impacted

Comment (style95) [reply core/nodejsActionBase/runner.js:49]: No edge case comes up in my mind too.

**Review by style95 — APPROVED** [review]
LGTM

**Comment by Luke-Roy-IBM** [comment]
Simple example that shows how the new functionality works

`index.js`
```javascript
function main(args) {
	const oneLinerJoke = require('./one-liner-joke');
	let getRandomJoke = oneLinerJoke.getRandomJoke();

	return {
        	body: getRandomJoke.body
    	}
}


module.exports.main = main;
```
`package.json`
```json
{
	"name": "my-action",
	"main": "index.js",
	"dependencies" : {
		"one-liner-joke" : "1.2.2"
 	}
}
```
